events/2021/12-contributor-celebration/README.md

```diff
@@ -0,0 +1,27 @@
+# 2021 Kubernetes Contributor Celebration
+
+## What
+
+An end of year online event to celebrate and get together with folks in the Kubernetes Contributor Community.
+
+## When and Where
+
+The dates of the Contributor Celebration are currently TBD, but will be held on Discord again, similar to last year. See https://www.kubernetes.dev/events/kcc2020/ for more information.
```

Comment:
Can you please update the link of 2020 Contributor Celebration to https://www.kubernetes.dev/events/past-events/2020/kcc2020/? The present link is a 404. Thank you.:relaxed:

Reply:
👍  shall update, good catch!

Reply:
Decided to remove the link because I haven't put in the information on the contributor site yet.